Sample/sdk-ios-swift/Profile.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Profile.swift
+//  sdk-ios-swift
+//
+//  Created by Simon Hamadene on 15/10/2018.
+//  Copyright © 2018 Luke Ashley-Fenn. All rights reserved.
+//
+
+import Foundation
+
+struct ProfileDictionary: Decodable {
+    let attributes: [Profile]
+}
+
+struct Profile {
+    let name: String
+    let value: String
+    
+    init(name: String, value: String) {
+        self.name = name
+        self.value = value
+    }
+}
+
+extension Profile: Decodable {
+    enum ProfileKeys: String, CodingKey {
+        case name
+        case value
+    }
+    
+    init(from decoder: Decoder) throws {
```

Comment:
I guess you can just use the default one?

Reply:
@casperkclee : Yes we can but I wanted to control it :-)